coral.py

```diff
@@ -664,6 +664,19 @@ def draw_tail(self, tail, direction):
 
         # Draw the rectangular part connecting to the next segment
         pygame.draw.circle(arena, tail_color, big_tail_center, 3 / 2* tail_radius)
+        
+    def draw(self):
+        # Draw the tail
+        for square in self.tail:
+            if square is self.tail[-1]:
+                if (len(self.tail) == 1):
+                    self.draw_tail(square, (self.xmov, self.ymov))
+                else:
+                    self.draw_tail(square, (self.tail[-2][0] - square[0], self.tail[-2][1] - square[1]))
+            else:
+                pygame.draw.rect(arena, HEAD_COLOR if self.alive else DEAD_HEAD_COLOR, square)
```

Comment:
I think it would make sense now to change HEAD_COLOR and DEAD_HEAD_COLOR to something like SNAKE_COLOR or BODY_COLOR, besides that lgtm

Reply:
That's right. The code would look better with this